the-blue-alliance-ios/View Elements/Awards/AwardCellViewModel.swift

```diff
@@ -14,7 +17,9 @@ struct AwardCellViewModel {
     init(award: Award) {
         awardName = award.name
         recipients = (award.recipients?.allObjects as? [AwardRecipient])?.map({ (recipient) -> Recipient in
-            return Recipient(teamKey: recipient.teamKey?.key, awardText: recipient.awardText)
+            let name = recipient.teamKey?.team?.nickname ?? recipient.teamKey?.team?.name
```

Comment:
I think what you're looking for here is either `recipient.teamKey?.name`

We can also force unwrap those team keys probably.